Source/include/Ably/ARTRealtimeChannel.h

```diff
@@ -168,6 +168,8 @@ ART_EMBED_INTERFACE_EVENT_EMITTER(ARTChannelEvent, ARTChannelStateChange *)
  */
 @property (nonatomic, readonly, nullable) NSString *channelSerial; // CP2b
 
+- (instancetype)initWithAttachSerial:(nullable NSString *)attachSerial channelSerial:(nullable NSString *)channelSerial;
```

Comment:
Can we document this to explain why it's exposed?

Reply:
done